src/Model/TransportModel/tsp1apt1.f90

```diff
@@ -76,7 +77,11 @@ module GwtAptModule
     integer(I4B), pointer :: ibudcsv => null() !< unit number for csv budget output file
     integer(I4B), pointer :: ncv => null() !< number of control volumes
     integer(I4B), pointer :: igwfaptpak => null() !< package number of corresponding this package
-    real(DP), dimension(:), pointer, contiguous :: strt => null() !< starting feature concentration
+    integer(I4B), pointer :: idxprepak => null() !< budget-object index that precedes package-specific budget objects
+    integer(I4B), pointer :: idxlastpak => null() !< budget-object index of last package-specific budget object
+    real(DP), dimension(:), pointer, contiguous :: strt => null() !< starting feature concentration (or temperature)
+    real(DP), dimension(:), pointer, contiguous :: ktf => null() !< thermal conductivity between the apt and groundwater cell
```

Comment:
Something seems off if we need to store thermal conductivity for the generic transport package class.

Reply:
Great catch, `ktf` should not have been in this...bad code merging.  Fixed in 7345130